pages/vi/dockertutorial.md

```diff
@@ -52,8 +52,25 @@ No matter your distribution of choice, you’ll need a 64-bit installation and a
 Please find your distro at https://docs.docker.com/install/#server and follow the guide to install Docker CE.
 
 
-Check your current Linux version with `uname -r` . You should see something like `3.10.[alphanumeric string].x86_64` or `4.19.43-microsoft-standard` for a windows based Linux system.
+Check your current Linux version with `hostnamectl` . Something like this will be in the terminal output:
 
+```  
+    Static hostname: <your_host_name_here>
+    Icon name: computer-laptop
+    Chassis: laptop
+    Machine ID: <your_machine_id_here>
+    Boot ID: <your_boot_id_here>
+    Operating System: Ubuntu 18.04.3 LTS 
+    Kernel: Linux 4.15.0-70-generic
+    Architecture: x86-64
+```  
+
+You can also use `cat /etc/os-release` or `lsb_release -a`.  
+Any of these three commands will tell you the information you need.
+
+You may need to use `uname -r`.  If using a windows based linux system.  You should see something like this `4.19.4-microsoft-standard`.
+
+For:
```

Comment:
there are a double FOR :) you can delete one.

Reply:
Yes there is, I will take care of that and get it resubmitted.